### DISCUSSION TITLE
Andoid immich app 1 album exlamation marks like album lost logical locatoin of photo on phone?

### DISCUSSION THREAD

**Discussion by sirweazel** [immich-app/immich#12538] [Q&A] [answered]
I have 1 album that somehow has broken. I'm not sure what is going on. The album is visible in web version fine. My other albums are visible.  There is 1 picture in the album that was previusly deleted on the device, so it shows in the album as cloud sync correctly.  All the other pictures on the device in that particular album show an exclamation mark.  Web version of the album contains thumbnails and actual photos.  It's like on the device, the link to the photos has been corrupted.  It's possible these photos where located in another location on the device, but i can't remember. When i was testing immich, i tested with a file folder album, and copied some photos to the location, then created an album.  Then i thought i removed those files, and re uploaded entire camera set of photos.  I can't state for sure that is what occurred.  What do you think is the best way to correct the issue, or how can i provide more information. I looked in the app for errors in the log, but didn't see any that appeared to this issue. 

Here are some sample screenshots.

If i open up photos app on the device, the photos are there and dispaly correctly.

Immich screenshots below
**Showing Album error**
![image](https://github.com/user-attachments/assets/46b36f9a-1ced-48ac-9efa-d315e4d1817e)
**Showing the 1 photo previously deleted from phone displays ok as cloud only**
![image](https://github.com/user-attachments/assets/707f4847-b9e2-4047-9c78-2adf6a41975e)
**Showing from the Camera synced album in immich are not displaying correctly**
![image](https://github.com/user-attachments/assets/e6eb628f-51a4-42c7-a724-bbfe7038920c)
**Immich web showing album is ok**
![image](https://github.com/user-attachments/assets/0fe10d26-8d03-427f-a3b4-868305ed662f)

**Comment by alextran1502** [comment, marked as answer]
Can you try log out and log back in the mobile app?

**Reply by sirweazel** [reply]
That fixed it! I probably should have tried that while troubleshooting. 
I'm curious why do you think that worked?
